docker/Dockerfile

```diff
@@ -10,7 +10,12 @@ RUN apt-get update && \
     . /opt/ros/humble/setup.sh && \
     rosdep update && \
     rosdep install --from-paths ${COLCON_WS}/src --ignore-src -r -y && \
-    colcon build --base-paths ${COLCON_WS}/src/ --build-base ${COLCON_WS}/build --install-base ${COLCON_WS}/install
+    colcon build --base-paths ${COLCON_WS}/src/ --build-base ${COLCON_WS}/build --install-base ${COLCON_WS}/install && \
+    rm -rf /var/lib/apt/lists/*
+
+# Env vars for the nvidia-container-runtime.
+ENV NVIDIA_VISIBLE_DEVICES all
```

Comment:
According to https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/user-guide.html

When using this ENV we should `--runtime=nvidia` option during `docker run`

Reply:
As you mentioned, the documentation states that `--runtime=nvidia` may be required. However, when using that option to run the container in my manjaro installation, it fails to start.
I then removed the `NVIDIA_VISIBLE_DEVICES` environment variable and without it, the simulation works both in Manjaro and Ubuntu. Removing this second variable, however, re-introduces the problem in Ubuntu.
The documentation does not say that you need `--runtime=nvidia` for `NVIDIA_VISIBLE_DEVICES`, so I guess that it should be fine to leave it out.

Reply:
Great, thanks for double checking that